example/reader_util.c

```diff
@@ -1745,6 +1745,32 @@ int ndpi_is_datalink_supported(int datalink_type) {
   }
 }
 
+struct vxlan_packet_header {
```

Comment:
I would suggest to use the already existing `struct ndpi_vxlanhdr` in `src/include/ndpi_typedefs.h` instead of redefining it.